File: HelperJobby/DTOs/Resume/CreateResumeDTO.cs
namespace HelperJobby.DTOs.Resume;

public class CreateResumeDTO
{
    public List<CreateUpdateEducationDTO> Educations { get; set; }
    public List<CreateUpdateWorkExperienceDTO> WorkExperiences { get; set; }
    public List<CreateSkillDTO> Skills { get; set; }
}

File: HelperJobby/client-app/src/DTOs/resumeRelatedDTOs/CreateResumeDTO.ts
import {CreateUpdateEducationDTO} from "./CreateUpdateEducationDTO";
import {CreateUpdateWorkExperienceDTO} from "./CreateUpdateWorkExperienceDTO";
import {CreateSkillDTO} from "./CreateSkillDTO";

export interface CreateResumeDTO {
    educations: CreateUpdateEducationDTO[];
    workExperiences: CreateUpdateWorkExperienceDTO[];
    skills: CreateSkillDTO[];
}